src/test/java/uk/gov/hmcts/ccd/domain/casestate/jexl/JexlEnablingConditionParserTest.java

```diff
@@ -71,7 +76,35 @@ void evaluateEnablingConditionWhenConditionContainsNotEqualityRegularExpression(
 
     @Test
     void evaluateEnablingConditionWhenConditionIsNull() {
-        Boolean isValid = this.enablingConditionParser.evaluate(null, createCaseData(
+        Boolean isValid = enablingConditionParser.evaluate(null, createCaseData(
+            "Test",
+            "I'm innocent"
+        ));
+
+        assertNotNull(isValid);
+        assertEquals(false, isValid);
+    }
+
+    @Test
+    void shouldReturnFalseWhenConditionHasUnKnownField() {
+        String enablingCondition = "FieldC!=\"*\" AND FieldB=\"I'm innocent\")";
```

Comment:
Maybe we could we change it to FieldC=\"*\" ? The opposite always resolves to false and how do we know test targets FieldC or !=\"*\"

Reply:
addressed